plugins/org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/ecore/EMFGeneratorFragment2.xtend

```diff
@@ -575,11 +575,16 @@ class EMFGeneratorFragment2 extends AbstractXtextGeneratorFragment {
 			genModel.runtimeVersion = emfRuntimeVersion
 			genModel.rootExtendsClass = 'org.eclipse.emf.ecore.impl.MinimalEObjectImpl$Container'
 			genModel.lineDelimiter = codeConfig.lineDelimiter
+			genModel.copyrightText = codeConfig.fileHeader.trimMultiLineComment
```

Comment:
This will throw if fileHeader is null which may be the case according to org.eclipse.xtext.xtext.generator.CodeConfig.initialize(Injector)